server/src/routes/coursePart.ts

```diff
@@ -8,55 +8,55 @@ import {processRequestBody} from 'zod-express-middleware';
 
 import {
   CourseRoleType,
-  EditAttainmentDataSchema,
-  NewAttainmentDataSchema,
+  EditCoursePartDataSchema,
+  NewCoursePartDataSchema,
 } from '@/common/types';
 import {
-  addAttainment,
-  deleteAttainment,
-  editAttainment,
-  getAttainments,
-} from '../controllers/attainment';
+  addCoursePart,
+  deleteCoursePart,
+  editCoursePart,
+  getCourseParts,
+} from '../controllers/coursePart';
 import {handleInvalidRequestJson} from '../middleware';
 import {courseAuthorization} from '../middleware/authorization';
 import {controllerDispatcher} from '../middleware/errorHandler';
 
 export const router = Router();
 
 router.get(
-  '/v1/courses/:courseId/attainments',
+  '/v1/courses/:courseId/course-parts',
```

Comment:
I think we could shorten this (and the other routes) to `/v1/courses/:courseId/parts`, similar to how we used to have `/v1/courses/:courseId/instances`.